packages/core/utils/package.json

```diff
@@ -49,15 +49,15 @@
     "@sindresorhus/slugify": "1.1.0",
     "date-fns": "2.30.0",
     "http-errors": "1.8.1",
-    "koa": "2.13.4",
-    "koa-body": "4.2.0",
     "lodash": "4.17.21",
     "p-map": "4.0.0",
     "yup": "0.32.9"
   },
   "devDependencies": {
     "@types/koa": "2.13.4",
     "eslint-config-custom": "*",
+    "koa": "2.13.4",
```

Comment:
Do we need to keep both koa & types/koa?

Reply:
Koa doesn't have typings so yes